src/core/mac/link_raw.hpp

```diff
@@ -64,6 +66,14 @@ class LinkRaw : public InstanceLocator
      */
     explicit LinkRaw(Instance &aInstance);
 
+    /**
+     * This method gets the associated `SubMac` object.
+     *
+     * @returns A reference to the `SubMac` object.
+     *
+     */
+    SubMac &GetSubMac(void) { return mSubMac; }
```

Comment:
Is it possible to move `SubMac` into `Instance`, so that I think it will be simpler to get this object.

Reply:
`SubMac` is intended to be used by  `Mac` or `LinkRaw` (which are its owner). It's not something that other objects would need to access.  So prefer to keep it in `Mac` and/or `LinkRaw` and not in `Instance`.